content/docs/manuals/app-catalog/nginx-server-unikraft.md

```diff
@@ -109,14 +116,21 @@ In this guide we'll use the Oakestra CLI tool to interact with the Oakestra plat
 
  {{< callout context="tip" title="Oakestra CLI Tool" icon="outline/rocket">}}
 
-In this guide we'll use the comprehensive Oakestra CLI toolkit to interact with the Oakestra-managed infrastructure. To find out more about the CLI tool, please refer to the CLI section of the manuals.
+In this guide we'll use the comprehensive Oakestra CLI toolkit to interact with the Oakestra-managed infrastructure. 
 
+{{< link-card
```

Comment:
good to know that you can in fact nest a link-card inside a callout - did not work for me when I tried it a few times 👀